test/test_selenium.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from selenium import webdriver
+from todo_app.data.session_items import create_trello_board,delete_trello_board
+from threading import Thread
+from todo_app.app import create_app
+
+import os
+@pytest.fixture(scope='module')
+def driver():
+ # path to your webdriver download
+    with webdriver.Chrome('./chromedriver') as driver:
+        yield driver
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+ # Create the new board & update the board idenvironment variable
+   board_id = create_trello_board()
+   os.environ['TRELLO_BOARD_ID'] = board_id
```

Comment:
You're setting this environment variable, but your program doesn't use it anywhere; currently your session items just always uses the first board that exists in its `getBoardid` function. That's potentially reasonable fall-back behaviour, but it does mean that we can't control the board that's selected without creating a fresh Trello account. 

We should make sure our tests aren't running against our main Trello board, e.g. by adding a check at the top of the `getBoardid` function that uses the environment variable `TRELLO_BOARD_ID` if it exists

Reply:
I was using this variable to delete the board in subsequent call. getBoardid is used for the original board, the test I wrote is not against the original board